### PR TITLE
Add a "remove plants" icon.

### PR DESCRIPTION
**Pull request by nic0lette**
This updates the ActionMode in "My garden" to include an action to
remove selected plants from the garden. It doesn't actually remove the
plants yet though.

Change-Id: I8e125596d06818257d5965d31409d30daba95920

### CODE REVIEW

File: app/src/main/res/menu/menu_garden_action_mode.xml [hunk 1]
```diff
@@ -15,6 +14,12 @@
   ~ limitations under the License.
   -->
 
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
-    <!-- TODO: Add ActionMode actions. -->
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
```
Comment (XinyueZ) [inline app/src/main/res/menu/menu_garden_action_mode.xml]: PTAL. 
NO. empty line between Line 18 - Line 19?

File: app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt [hunk 2]
```diff
@@ -124,10 +124,15 @@ class GardenFragment : Fragment() {
      * Inner class to encapsulate handling of the [ActionMode] in my garden.
      */
     private inner class GardenActionModeCallback : ActionMode.Callback {
-        override fun onActionItemClicked(mode: ActionMode, item: MenuItem): Boolean {
-            // TODO: Implement ActionMode actions.
-            return false
-        }
+        override fun onActionItemClicked(mode: ActionMode, item: MenuItem): Boolean =
```
Comment (tiembo) [inline app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt]: can omit Boolean

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -32,6 +32,7 @@
     <string name="planted_date" translation_description="Text with placeholders that indicates when the plant was planted. Example: 'Sunflower planted on May 24, 2018'.">%1$s planted on %2$s</string>
     <string name="menu_item_share_plant" translation_description="The label for the Share menu item">Share</string>
     <string name="share_text_plant" translation_description="The default messaged used when sharing a plant on SNS, etc.">Check out the %s plant in the Android Sunflower app</string>
+    <string name="remove_plant_title" translation_description="Title of menu item for removing a plant from one's garden">Remove plants</string>
```
Comment (tiembo) [inline app/src/main/res/values/strings.xml]: Change to "Remove plant(s)"

File: app/src/main/res/menu/menu_garden_action_mode.xml (hunk 1 above)
Comment (tiembo) [reply app/src/main/res/menu/menu_garden_action_mode.xml]: +1 for newline to match other menu XML files

Comment (nic0lette) [reply app/src/main/res/menu/menu_garden_action_mode.xml]: Done. :)

File: app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt (hunk 2 above)
Comment (nic0lette) [reply app/src/main/java/com/google/samples/apps/sunflower/GardenFragment.kt]: Done.

File: app/src/main/res/drawable/ic_delete_24dp.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<!--
```
Comment (XinyueZ) [inline app/src/main/res/drawable/ic_delete_24dp.xml]: NAK.
ic_delete_24dp.xml which `24dp` .

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (nic0lette) [reply app/src/main/res/values/strings.xml]: Done.

File: app/src/main/res/drawable/ic_delete_24dp.xml (hunk 4 above)
Comment (nic0lette) [reply app/src/main/res/drawable/ic_delete_24dp.xml]: Sorry, I'm not sure I understand.

Comment (XinyueZ) [reply app/src/main/res/drawable/ic_delete_24dp.xml]: sorry,  🤕 my bad.
I mean the file name is strange that it is with "24dp" .

Comment (tiembo) [reply app/src/main/res/drawable/ic_delete_24dp.xml]: I think it's OK.  The filter icon also has a "24dp" suffix.

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -26,7 +26,7 @@
     <string name="planted_date">%1$s gepflanzt in %2$s</string>
     <string name="menu_item_share_plant">Teilen</string>
     <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
-    <string name="remove_plant_title">Pflanze(n) entfernen</string>
+    <string name="remove_plant_title">Pflanze (n) entfernen</string>
```
Comment (XinyueZ) [inline app/src/main/res/values-de/strings.xml]: I think in German, people still like using without empty `Pflanze(n)`, I feel at least very strange to see empty there.

Comment (nic0lette) [reply app/src/main/res/values-de/strings.xml]: Fixed. Yeah, I .. don't know why I trusted Google Translate for that ;) Thanks!

**Pull request merged**